CIP-XXXX/README.md

```diff
@@ -0,0 +1,315 @@
+6. The project API returns a documented status code indicating the success or failure of the operation
+7. If a successful status is detected and returned, the project issues tokens to the specified address per their campaign settings
+
+## URI Format
+
+The URI format consists of the CIP-13 `web+cardano://` scheme, followed by the `claim` authority, then a `version` path.
+
+***NOTE: ALL ARGUMENTS SHOULD BE URL-ENCODED***
+
+### Version 1
+
+Version 1 URIs must include `/v1` as the path of the URI.
+
+Version 1 URIs must include two required arguments:
+
+* `faucet_url` as a fully-typed URL (i.e. https://claim.hosky.io)
+* `code` as either a campaign identifier or unique, one-time use code
+
+_Version 1 Examples:_
+
+```html
+<!-- A Version 1 Cardano Claim URI with campaign identifier code -->
+<a href="web+cardano://claim/v1?faucet_url=https%3A%2F%2Fclaim.hosky.io&code=consensus2023">Thanks for attending Consensus 2023!</a>
+
+<!-- A Version 1 Cardano Claim URI with unique, one-time use code -->
+<a href="web+cardano://claim/v1?faucet_url=https%3A%2F%2Fclaim.hosky.io&code=dff6508d8dfb4e128fd67e9ff54af147">Claim your $HOSKY now!</a>
+```
+
+### Version 2
+
+Version 2 URIs must include `/v2` as the path of the URI
+
+Version 2 URIs must include one required argument:
+
+* `faucet_url` as a fully-typed URL (i.e. https://claim.hosky.io)
+
+Version 2 URIs may include additional, optional arguments:
+
+* `code` as either a campaign identifier or unique, one-time use code
+* Additional arguments should be passed by the wallet to the API server in the JSON body of the POST request.
+
+
+_Version 2 Examples:_
+```html 
+<!-- A Cardano Claim URI with only the URL specified -->
+<a href="web+cardano://claim/v2?faucet_url=https%3A%2F%2Fclaim.hosky.io">Claim $HOSKY Now!</a>
+
+<!-- A Cardano Claim URI with a campaign-specific code -->
+<a href="web+cardano://claim/v2?faucet_url=https%3A%2F%2Fclaim.nftxlv.com&code=NFTxLV2023">Claim your NFTxLV 2023 NFT now!</a>
+
+<!-- A Cardano Claim URI with a unique, one-time use code -->
+<a href="web+cardano://claim/v2?faucet_url=https%3A%2F%2Fclaim.hosky.io&code=ABC123">Claim Some $HOSKY!</a>
+
+<!-- A Cardano Claim URI with a campaign-specific code and optional user_id -->
+<a href="web+cardano://claim/v2?faucet_url=https%3A%2F%2Fclaim.hosky.io&code=NFTxLV2023&user_id=Idjiot1337">Get your $HOSKY!</a>
+```
+
+## Wallet Requests
+
+Light wallets that detect and support `web+cardano` URIs as well as mobile wallets who detect either a QR code or other
+link with this format should parse the URI and send a `POST` request to the specified URL containing a JSON payload
+including:
+
+* The user's wallet receive address
+* The code (if present in the URI [required for Version 1])
+
+_Examples:_
+```json 
+URI: web+cardano://claim/v1?faucet_url=https%3A%2F%2Fclaim.hosky.io
+URL: https://claim.hosky.io
+POST JSON Data:
+{
+  "address": "addr1abc...xyz"
+}
+
```

Comment:
I tried preparing a commit which goes something like this: but it changes the appearance too radically for me to push it into your branch.  Can you do something like this? ... preserving readability as you have imagined it, and being enough to dismiss the ugly red highlighting of the currently reported JSON syntax errors here.

---

URIs:
```
web+cardano://claim/v1?faucet_url=https%3A%2F%2Fclaim.hosky.io
https://claim.hosky.io
```
POST JSON Data:
```json 
{
  "address": "addr1abc...xyz"
}
```
etc...

Reply:
Thanks Robert! I will take a look and make the modifications necessary per your feedback :)

Reply:
This should be resolved in the latest commit.

Reply:
@Crypto2099 looks better all around 🤓